package.json

```diff
@@ -169,7 +170,7 @@
         "translation:fetch": "node scripts/handle-translations.js pull-master",
         "translation:branch": "node scripts/handle-translations.js branch",
         "translation:check": "cd scripts && node extract-translations.js show-untranslated && cd ..",
-        "branch:create": "node scripts/branch-manager.js",
+        "branch:create": "node script/branch-manager.js",
```

Comment:
Is this allright, we're changing from  `node scripts` to `node script`

Reply:
@amir-deriv i think it should be revert back to `scripts`